src/mc_tasks/lipm_stabilizer/StabilizerTask.cpp

```diff
@@ -1142,15 +1142,19 @@ void StabilizerTask::updateFootForceDifferenceControl()
     return;
   }
 
+  sva::PTransformd T_0_L(leftFootTask->surfacePose().rotation());
+  sva::PTransformd T_0_R(rightFootTask->surfacePose().rotation());
```

Comment:
I start to consider these lines should be
```
sva::PTransformd T_0_L = leftFootTask->surfacePose();
sva::PTransformd T_0_R = rightFootTask->surfacePose();
```
for wrench transformation although we used only the rotation in the similar case https://github.com/arntanguy/lipm_walking_controller/pull/2/files#diff-86694886f91b285c323e710f1b64ec1c0f5ab403d0ee748e112030f766fcae11R139

I'm not confident, so could anyone give some idea?

Reply:
I reconsidered and came to the conclusion that it would be OK to leave it as it is ("as is" in the merged PR).
For wrench, only force is used (moment is not used), so the position of the transformation is not important.
For velocity, the angular velocity is set to zero as shown below, which implicitly assumes a frame whose position is identical to the surface and whose orientation is identical to the inertial frame.  In such cases, it is correct to apply rotation-only transformations.
https://github.com/jrl-umi3218/mc_rtc/pull/290/files/72f45b20f3766351b296e37b734b65113184baeb#diff-be940a972e16470195608839479a680e1f8aba715c2e5f4c62cdaa4666be420eR1169-R1170